windows/README.md

```diff
@@ -98,10 +98,50 @@ the ```driver/src/winmonitor_gen.c``` file.
 Finally, rebuild the driver. Note that new OS versions may considerably change the kernel structure, requiring upgrading
 the generation scripts.
 
+Code Coverage for Windows Binaries
+==================================
+
+Binaries produced by Microsoft tools contain line information in PDB files. These files have a proprietary
+format and are not readable by Linux tools. This poses several challenges in order to covert program counters
+produced by S2E's coverage plugin into say LCOV reports.
+
+The following instructions explain how to get LCOV coverage reports for Windows binaries.
+
+- Build the solution in Visual Studio
+
+- Run ```pdbparser.exe``` as follows:
```

Comment:
Note that in Markdown, inline code only needs a single back-tick around it. Triple back-ticks are typically used for code blocks (see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)